app-connector-client/test/test_integration.js

```diff
@@ -1,6 +1,6 @@
 
 var request = require('supertest');
-var server = require("../server/app")("test/varkes_config.js");
+var server = require("../server/app")("test/varkes_config.json");

```

Comment:
Are you solving a different ticket here (the one about transforming config into a json)? Please avoid that in future to simplify life of writer and reviewer.